frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_language_view.dart

```diff
@@ -12,85 +14,86 @@ class SettingsLanguageView extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return SingleChildScrollView(
-      child: Column(
-        crossAxisAlignment: CrossAxisAlignment.start,
-        children: [
-          Row(
-            children: [
-              FlowyText.medium(LocaleKeys.settings_menu_language.tr()),
-              const LanguageSelectorDropdown(),
-            ],
-          ),
-        ],
+      child: BlocBuilder<AppearanceSettingsCubit, AppearanceSettingsState>(
+        builder: (context, state) => Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Row(
+              children: [
+                Expanded(
+                  child:
+                      FlowyText.medium(LocaleKeys.settings_menu_language.tr()),
```

Comment:
Maybe a comma for fomatting after `tr(),`

Reply:
there's only one parameter. I don't think the analyzer would complain.

Reply:
I know, it's not about the number of parameters only though, it's also about line length. This was more of a personal preference, hence the maybe at the beginning. 🤣👍